### ISSUE TITLE
Zero Output at MAX30100_RawData and MAX30100_Minimal

### ISSUE THREAD

**Issue by nazmibojan** [oxullo/Arduino-MAX30100#28]
### Troubleshooting checklist

- [OK] I read the README (on master) thoroughly
- [OK] I ran the MAX30100_Tester and I'm going to paste the output down below
- [OK] I filled in all the details of my setup down below

### Description of the issue
Hi @oxullo,
I have run MAX30100_Tester and got output like below. I have been wondered why is the temperature show 0C. After that I run MAX30100_Manual and MAX30100_RawData. I got zero output in RawData and in Manual that show 0 of bpm dan 0% of SpO2. So can you help me where I start to solve my problem?

![max30100 manual](https://user-images.githubusercontent.com/17125575/30538093-92266138-9c96-11e7-9249-693263e194cd.PNG)

![raw data max30100](https://user-images.githubusercontent.com/17125575/30538101-99991672-9c96-11e7-9453-8c909ddba319.PNG)

 
### Output from MAX30100_Tester example

![max30100 tester](https://user-images.githubusercontent.com/17125575/30538127-b222e92a-9c96-11e7-84a0-baca375c909b.PNG)

### Details of my setup

* Arduino hardware: Arduino Uno R3
* MAX30100 breakout: https://www.tokopedia.com/caravane/max30100-pulse-oximeter-heart-rate-sensor-module
* Arduino framework version: Arduino IDE 1.8.4
* MAX30100 library version: v1.1.0

**Comment by oxullo** [comment]
First thing that comes in my mind: check if the I2C voltage shifter is able to work at 400kHz

**Comment by nazmibojan** [comment]
Oh OK, it make sense. I have to find my shifter datasheet. But may I know where was the 400kHz came from?

**Comment by oxullo** [comment]
https://github.com/oxullo/Arduino-MAX30100/blob/4550b2d1753ba00e4330afb763c908eb9a133f76/src/MAX30100.cpp#L30 and https://github.com/oxullo/Arduino-MAX30100/blob/4550b2d1753ba00e4330afb763c908eb9a133f76/src/MAX30100.h#L33

MAX30100 can operate in full speed mode hence the library initializes I2C in full speed.

**Comment by liviuanc** [comment]
the same problem

**Comment by liviuanc** [comment]
And I lower the i2c speed

**Comment by nazmibojan** [comment]
How fast is your I2C speed now @liviuanc ?

**Comment by oxullo** [comment]
@nazmibojan have you had any success by lowering the clock speed?

**Comment by nazmibojan** [comment]
@oxullo No. I still had the same problem.

**Comment by oxullo** [comment]
@nazmibojan does the sensor shine red during the test run? is the arduino you have a genuine one? have you tried with a 3.3V microcontroller?
My baseline test setup is very similar to yours and it works, but an alternative that I'm testing right now is to pull up SDA and SCL to 5V. It works so far but since it worked also with 3.3V I can't properly compare.

**Comment by AhmadZMQ** [comment]
**@oxullo**  I am using Nano and MAX30100 sensor and their connection is -pullup SDA and SCL with 4.7k ohm to the 5V. There after he got working but HB, Spo2 and  temp show zero.Also the RED led does not light up normally ,it does when connected with GND alone (not with IR LED).
After uploading all thing is correct but values gives zero.
What i do to avoid it?

**Comment by oxullo** [comment]
@AhmadZMQ please open a new issue and make sure you fill in the details requested in the issue template

**Comment by nazmibojan** [comment]
Hi @oxullo , my MAX30100 doesn't shine red during the test. Why did it happen?
I don't know about  genuineness of my Arduino, it's a gift from my friend years ago. I have been tried using STM32F1 development board, it shows the right temperature but I don't continue this project because there is another important business that i must finished.

**Comment by oxullo** [comment]
@nazmibojan whenever you want to get back on track on this project, please open a new ticket and provide a sketch of how you connected. Your issue is similar to the one @AhmadZMQ is experiencing, so eventually follow #33 beforehand.

**Comment by nazmibojan** [comment]
Finally, I just found the problem of my MAX30100 module. Actually, I feel strange when I look back to my MAX30100 price. That's too cheap in my country than amazon or another online store outside Indonesia. I found Pin 1 (Vout) of 3.3V regulator didn't connect to Pin 2 (Vin) of 1.8V regulator. A Via that connects that two pin didn't has a hole (that means a via didn't success to be created between top and bottom layer). Oh God, that's very bad product. Below I attached my result for MAX30100_Tester.
Thank's all.

![max30100 tester](https://user-images.githubusercontent.com/17125575/33203378-5edd4fc8-d132-11e7-87ce-5bb5092012ea.PNG)
